utils/mcp_client.py

```
@@ -0,0 +1,677 @@
+    env: Mapping[str, str] | None = None
+    cwd: str | None = None
+    client_name: str = "speaking-meeting-bot"
+    client_version: str = "0.1.0"
+    _state: _JsonRpcState = field(default_factory=_JsonRpcState, init=False)
+    _process: asyncio.subprocess.Process | None = field(default=None, init=False)
+
+    async def start(self) -> None:
+        if self._process is not None:
+            return
+        if not self.command:
+            raise McpClientError("Stdio MCP command cannot be empty")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)